tigerpath/majors_and_certificates/majors/AAS.json

```diff
@@ -45,8 +31,9 @@
           "max_counted": null,
           "min_needed": 1,
           "course_list": [
+            "AAS 244",

```

Comment:
https://aas.princeton.edu/academics/undergraduate-program/concentration-requirements

"AAS 244 Introduction to Pre-20th Century Black Diaspora Art (Pre-20th century)
AAS 245 Introduction to 20th Century African American Art
AAS 353 African American Literature: Origins to 1910 (Pre-20th century)
AAS 359 African American Literature: Harlem Renaissance to Present
AAS 366 African American History to 1863 (Pre-20th century)
AAS 367 African American History Since Emancipation"